eclair-core/src/test/scala/fr/acinq/eclair/payment/PostRestartHtlcCleanerSpec.scala

```diff
@@ -543,6 +544,105 @@ class PostRestartHtlcCleanerSpec extends TestKitBaseClass with FixtureAnyFunSuit
     eventListener.expectNoMsg(100 millis)
   }
 
+  test("Relayed nonstandard->standard HTLC is retained") { f =>
+    import f._
+
+    val relayedPaymentHash = randomBytes32
+    val trampolineRelayedPaymentHash = randomBytes32
+    val trampolineRelayed = Origin.TrampolineRelayedCold((channelId_ab_1, 0L) :: Nil)
+    val relayedHtlc1In = buildHtlcIn(0L, channelId_ab_1, trampolineRelayedPaymentHash)
+    val relayedhtlc1Out = buildHtlcOut(0L, channelId_ab_2, trampolineRelayedPaymentHash)
+    val nonRelayedHtlc2In = buildHtlcIn(1L, channelId_ab_1, relayedPaymentHash)
+
+    val pluginParams = new CustomCommitmentsPlugin {
+      def name = "test with incoming HTLC from remote"
+      def getIncomingHtlcs: Seq[PostRestartHtlcCleaner.IncomingHtlc] = List(PostRestartHtlcCleaner.IncomingHtlc(relayedHtlc1In.add, None), PostRestartHtlcCleaner.IncomingHtlc(nonRelayedHtlc2In.add, None))
+      def getHtlcsRelayedOut(htlcsIn: Seq[PostRestartHtlcCleaner.IncomingHtlc]): Map[Origin, Set[(ByteVector32, Long)]] = Map.empty
+    }
+
+    val nodeParams1 = nodeParams.copy(pluginParams = List(pluginParams))
+    val c = ChannelCodecsSpec.makeChannelDataNormal(List(relayedhtlc1Out), Map(50L -> trampolineRelayed))
```

Comment:
Why did you change the htlcId to 50? It doesn't make any sense now...this one is supposed to map `relayedHtlc1Out` to `relayedHtlc1In` so it must be `0L -> trampolineRelayed`.

Reply:
Strange, I thought that number is local HTLC id in a channel so i can differ from HTLC id in origin. Also wonder why the test still works.

Reply:
OK, that number does not matter here, it's used at https://github.com/ACINQ/eclair/blob/83b50347447266537426f905affb656cfb1e17bc/eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/PostRestartHtlcCleaner.scala#L362 to match `htlcsToIgnore` which are empty in this test.